code/modules/antagonists/traitor/objectives/final_objective/supermatter_cascade.dm

```diff
@@ -9,7 +9,11 @@
 	var/sent_crystal = FALSE
 
 /datum/traitor_objective/final/supermatter_cascade/generate_objective(datum/mind/generating_for, list/possible_duplicates)
-	if(!can_take_final_objective())
+	var/valid_crystal = FALSE
+	for(var/obj/machinery/power/supermatter_crystal/engine/crystal in GLOB.machines)
+		if(isturf(crystal.loc) || (is_station_level(crystal.z) || is_mining_level(crystal.z)))
```

Comment:
why would isturf be false

Reply:
I was _referencing_ code from the CIMS program and that must have been left in. My bad.